app/javascript/packs/document-capture.tsx

```diff
@@ -32,6 +32,8 @@ interface AppRootData {
   cancelUrl: string;
   idvInPersonUrl?: string;
   securityAndPrivacyHowItWorksUrl: string;
+  inPersonCtaVariantTestingEnabled: string;
```

Comment:
should this be a boolean to match the type in `/document-capture/context/in-person.ts`?

Reply:
oh, are does this need to be a string bc of how it's used in appRootData later on?

Reply:
That seems right to me. Change made in latest PR.